components/camel-platform-http-vertx/src/generated/resources/platform-http-vertx.json

```diff
@@ -0,0 +1,12 @@
+{
+  "other": {
+    "kind": "other",
```

Comment:
Shouldn't this be a component?

Reply:
it is not a component, i.e. you can't use it in a `from(...)`

Reply:
Thanks for the explanation!